client/src/App.css

```diff
@@ -11,6 +11,36 @@
   pointer-events: none;
 }
 
+.playerInputEdit {
```

Comment:
@gantgarrett remember to use ChakraUI for CSS rather than vanilla CSS wherever possible. 

And if ChakraUI doesn't have support for some CSS you want to use, then make a new folder in the same directory as the component and name it the same as the component. Then you can drag the component into that folder and create a new .css file inside that component where you can write any css that ChakraUI doesn't have support for. That way the CSS is scoped to the component.

Reply:
Hey Michael,

Made a commit with work changes discussed yesterday on the edit buttons. Using the Chakra UI icon library to use the edit icon and the icons' color corresponds to the team color. Putting a border around the buttons looked weird so I just have the icon colors. Let me know what you think!